metrics/density/footprint_data.sh

```diff
@@ -167,16 +167,16 @@ function get_proc_pss() {
 function grab_vm_uss() {
 	proxy=$(get_proc_uss $PROXY_PATH)
 	shim=$(get_proc_uss $SHIM_PATH)
-	qemu=$(get_proc_uss $HYPERVISOR_PATH)
+	hypervisor=$(get_proc_uss $HYPERVISOR_PATH)
 	virtiofsd=$(get_proc_uss $VIRTIOFSD_PATH)
 
-	total=$((proxy + shim + qemu + virtiofsd))
+	total=$((proxy + shim + hypervisor + virtiofsd))
 
 	local json="$(cat << EOF
 		"uss": {
 			"proxy": $proxy,
 			"shim": $shim,
-			"qemu": "$qemu",
+			"$KATA_HYPERVISOR": "$hypervisor",
```

Comment:
It uses `KATA_HYPERVISOR` variable on which a default value isn't declared in this script nor it seems to be in the sourced `common.bash`. I may be missing something (first time looking at this script), but wouldn't it need to define a default somewhere to be safe?

Reply:
@wainersm it is declared here https://github.com/kata-containers/tests/blob/4fede8941f5ad6b820652eef1227681044429cbf/lib/common.bash#L28 and https://github.com/kata-containers/tests/blob/4fede8941f5ad6b820652eef1227681044429cbf/lib/common.bash#L145

Reply:
you can see that the variable can have multiple definitions

Reply:
if you are running a different hypervisor like `firecracker` or `clh` you need to setup the `KATA_HYPERVISOR` variable so it can perform the setup and install the correspondent hypervisor

Reply:
@GabyCT ah, ok, I realized I look at `metrics/lib/common.bash` only.

OK, this PR looks good to me.